render/go/renderer.go

```diff
@@ -129,3 +126,52 @@ func isHttpStatusCode(s string) bool {
 	}
 	return false
 }
+
+func NewGinServerTestConfig() config.Data {
+	config.ConfigFilePath = "../render/go/config_goginserver_test.yml"
+	config.Debug = true
+	return config.Data{
+		Template: config.Template{
+			Name: "go-gin",
+		},
+		OpenAPIFile: "../openapi/test/fixtures/validation_no_refs.yaml",
+		OutputPath:  ".",
+	}
+}
+func NewGoClientTestConfig() config.Data {
+	config.ConfigFilePath = "../render/go/config_goclient_test.yml"
+	config.Debug = true
+	return config.Data{
+		Template: config.Template{
+			Name: "go-client",
+		},
+		OpenAPIFile: "../openapi/test/fixtures/validation_no_refs.yaml",
+		OutputPath:  ".",
+	}
+}
```

Comment:
Do these have to be here, or can they move to the appropriate `_test.go` file?

Reply:
we can't import from a `*_test.go` file and these are needed in multiple places. I originally had them in the test file and would rather not create a `helper` file.